static/operator.yaml

```diff
@@ -47,7 +49,8 @@ metadata:
   namespace: istio-operator
   name: example-istiocontrolplane
 spec:
-  profile: default
+  profile: demo
+---
```

Comment:
if the plan is to terminate each file in istio/operator/deploy with `---` you should remove line 65.  This has impact on the e2e tests too.  Please point out the PR where the deploy directory is changed so I can make appropriate changes in the e2e tests.